scripts/rfid_trigger_play.py

```diff
@@ -0,0 +1,714 @@
+# file:
+# settings/rfid_trigger_play.conf
+# Please consult this file for more information.
+# Do NOT edit anything in this file.
+
+####################################################
+# VARIABLES
+
+# ignore files will these extensions in the results:
+ignore_file_extension = ('.conf', '.ini', '.jpg', '.db', '.dat', '.*~')
+
+# The absolute path to the folder which contains all the scripts
+path_current_dir_absolute=str(pathlib.Path(__file__).parent.absolute())
+
+# config file location
+path_dir_root = os.path.abspath(path_current_dir_absolute + "/..")
+path_dir_audiofolders = os.path.abspath(path_dir_root + "/shared/audiofolders")
+path_dir_settings = os.path.abspath(path_dir_root + "/settings")
+path_file_debuglog = os.path.abspath(path_dir_root + "/logs/debug.log")
+path_config_global = os.path.abspath(path_dir_settings + "/global.conf")
+path_config_debug = os.path.abspath(path_dir_settings + "/debugLogging.conf")
+path_config_rfid = os.path.abspath(path_dir_settings + "/rfid_trigger_play.conf")
+path_txt_latestID = os.path.abspath(path_dir_root + "/shared/latestID.txt")
+path_dir_shortcuts = os.path.abspath(path_dir_root + "/shared/shortcuts")
+path_dir_playlists = os.path.abspath(path_dir_root + "/playlists")
+# read three config files into one dictionary
+conf = conf_read_bash([path_config_global, path_config_debug, path_config_rfid])
+
+###################################
+# parse variables from command line
+parser = argparse.ArgumentParser()
+parser.add_argument('-i', '--cardid')
+parser.add_argument('-c', '--command')
+parser.add_argument('-d', '--dir')
+parser.add_argument('-v', '--value')
+args = parser.parse_args()
+args_main = vars(args)
+logger.debug('arguments passed on to script:')
+logger.debug(args_main)
+
+# Set the date and time of now
+now_string = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+
+# create the configuration file from sample - if it does not exist
+if Path(path_config_rfid).is_file():
+    # file exists
+    logger.debug("rfid_trigger_play.conf exists")
+else:
+    # file does NOT exist
+    logger.debug("rfid_trigger_play.conf does NOT exist, copying from settings dir")
+    copyfile(path_dir_settings + "/rfid_trigger_play.conf.sample", path_config_rfid)
+
+# Read config files for global, debug and rfid
+conf = {}
+# global config
+with open(path_config_global) as myfile:
+    for line in myfile:
+        if not line.lstrip().startswith('#'):
+            name, var = line.partition("=")[::2]
+            conf[name.strip()] = var.strip()
+# debugging config
+with open(path_config_debug) as myfile:
+    for line in myfile:
+        if not line.lstrip().startswith('#'):
+            name, var = line.partition("=")[::2]
+            conf[name.strip()] = var.strip()
+# debugging rfid
+with open(path_config_rfid) as myfile:
+    for line in myfile:
+        if not line.lstrip().startswith('#'):
+            name, var = line.partition("=")[::2]
+            conf[name.strip()] = var.strip()
```

Comment:
You are repeating yourself a lot here. you could write a helper function to read a file based on a given path